cluster/health-monitor.sh

```diff
@@ -27,13 +32,14 @@ CHECK_PERIOD=${CHECK_PERIOD:-10}
 # and containerd.
 SLEEP_SECONDS=${SLEEP_SECONDS:-120}
 
+sleep ${INITIAL_WAIT_SECONDS}
```

Comment:
readiness_check()

Reply:
Good point. We may not want to let this script rely on another script. So migrate the code here.